wallet/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
     implementation 'org.greenrobot:eventbus:3.1.1'
     implementation 'com.tbuonomo.andrui:viewpagerdotsindicator:4.1.2'
+    implementation 'com.sambarboza:orbitview:0.1.4'
```

Comment:
I am not sure why this isn't showing up on jcenter().  Instead I added this to the `repositories` section of `wallet/build.gradle`

```
    maven {
        url 'https://dl.bintray.com/sambarboza/orbitview'
        name 'orbitview'
    }
```

Do we have to wait for this library to be published?